src/components/pagination/pagination.stories.tsx

```diff
@@ -30,7 +30,7 @@ export const Primary: Story = {
           pageSize={pageSize}
           activePage={page}
           totalItems={200}
-          totalPages={10}
+          totalPages={100}
```

Comment:
Can we also adjust the total items count for default page size to make the story consistent for viewers?